src/StartMenu.ts

```diff
@@ -261,6 +263,8 @@ export class AEntryPointList implements IEntryPointList {
 
   protected data: INamedSet[] = [];
 
+  private extensionFilters: (p: {[key: string]: any}) => any;
```

Comment:
why is the return type `any` shouldn't it be `boolean`? + please use a better parameter name than `p`